rasa/model.py

```diff
@@ -65,17 +67,19 @@ def get_model(model_path: Text = DEFAULT_MODELS_PATH) -> Optional[UnpackedModelP
         Path to the unpacked model.
 
     """
-    if not model_path:
-        return None
-    elif not os.path.exists(model_path):
-        return None
-    elif os.path.isdir(model_path):
+    if not model_path or not os.path.exists(model_path):
+        raise ModelNotFound()
+
+    if os.path.isdir(model_path):
         model_path = get_latest_model(model_path)
+    elif not model_path.endswith(".tar.gz"):
+        # `model_path` does not point to a directory or a .tar.gz
+        model_path = None
 
-    if model_path:
-        return unpack_model(model_path)
+    if not model_path:
+        raise ModelNotFound()
```

Comment:
can we add some information to the exception from which path it tried to load the model?

Reply:
(also to the other locations where we raise this exception)

Reply:
Should I include just the path or an additional message?
```python
raise ModelNotFound(model_path)
raise ModelNotFound("Path '{}' does not point to a .tar.gz file.".format(model_path))
```

Reply:
Went with the second option.